modules/ROOT/pages/about-components.adoc

```diff
@@ -1,57 +1,176 @@
++
+image::components-core-studio.png[Core Components in Studio]
++
+Notice that the components are subdivided into types, including Batch,
+Error Handling, and Flow Control.
++
+* In Design Center, when you are building a Mule app, you can find Mule
+components listed among *Modules* in the *Select a Component* dialog.
++
+image::components-core-fd.png[Core Components in Design Center]
++
+Design Center provides many of the Core components described below. Though the
+Design Center UI does not subdivide components into the types you see in the
+Studio UI, it can help to conceptualize them by those types.
 
-Mule apps mimic tradition procedural programming, with the advantage that data mapping is replaced with DataWeave scripts (or visual mapping) instead of complex Java classes and code. However, the core building blocks are the same.
+== Batch
 
-//, and the product language should align with Studio.
+To process Mule messages in batches, instead of processing them all together,
+you can use Batch components, which include:
 
-Mule apps are built by combining flows together through connectors (or in the case of Studio, through private flows and sub-flows).
+* Batch Aggregator
+* Batch Job
+* Batch Step
 
-Connectors and Schedulers trigger flows by passing in an event. Other components then transform and process the event though a sequence of linked steps.
+See xref:batch-processing-concept.adoc[Batch Processors].
+
+== Components
 
-These event processing components include individual event processors like a Logger or Transform component, or other outbound connectors that request additional data to be merged with the existing event later in the flow.
+These Core components perform a variety of tasks:
 
-To help combine and transform data, event processors can be grouped together to perform other common processing tasks. These groupings are categorized as Routers or Scopes.
+* xref:business-events-custom.adoc[Custom Business Events]: For collecting information about about flows and message processors that handle your business transactions. See also, xref:business-events.adoc[Business Events].
+* xref:dynamic-evaluate-component-reference.adoc[Dynamic Evaluate]: For dynamically selecting a script, instead of hardcoding it through the Transform Message Component.
+* xref:flowref-about.adoc[Flow Reference]: For routing the Mule event to another
+flow or subflow (and back) within a Mule app.
+* xref:logger-component-reference.adoc[Logger]: For logging important information about
+your Mule app, such as error messages and status notifications.
+* xref:parse-template-reference.adoc[Parse Template]: For processing a template
+and obtaining a result.
+* xref:transform-component-about.adoc[Transform Message]: For converting input
+data to a new output structure or format.
 
-A Scope (blue color in Design Center) groups a sequence of event processors together to apply some programming behavior to that isolated sequence of event processors.
+== Endpoints
```

Comment:
Is this really under `Endpoints`? We got rid of this concept in Mule 4 and the `scheduler` is considered a "source".

Reply:
yes, under Endpoints in the Studio UI unfortunately